sklearn/_loss/loss.py

```diff
@@ -583,7 +584,7 @@ class PinballLoss(BaseLoss):
     Additional Attributes
     ---------------------
     quantile : float
-        The quantile to be estimated. Must be in range (0, 1).
+        The quantile level of the quantile to be estimated. Must be in range (0, 1).
```

Comment:
```suggestion
        The quantile level to be estimated. Must be in range (0, 1).
```

Reply:
No. It is very intentional as it is. The quantile is a number such that the probability to be smaller than that number equals x. Now, how do you call x? It is the probability level of the quantile, or just level (often called alpha).

I have to open an issue to fix my mistakes of the past.